src/423groupby.js

```diff
@@ -319,41 +300,17 @@ if(false) {
 						if (col.expression.columnid === '*') {
 							return pre + "g['" + colas + "']++;" + post;
 						} else {
-							return (
-								pre +
-								'if(typeof ' +
-								colexp +
-								'!="undefined") g[\'' +
-								colas +
-								"']++;" +
-								post
-							);
+							return pre + 'if(typeof ' + colexp + '!="undefined") g[\'' + colas + "']++;" + post;
 						}
 					} else if (col.aggregatorid === 'ARRAY') {
 						return pre + "g['" + colas + "'].push(" + colexp + ');' + post;
 					} else if (col.aggregatorid === 'MIN') {
 						return (
-							pre +
-							"g['" +
-							colas +
-							"']=Math.min(g['" +
-							colas +
-							"']," +
-							colexp +
-							');' +
-							post
+							pre + 'if ((y=' + colexp + ") < g['" + colas + "']) g['" + colas + "'] = y;" + post
 						);
 					} else if (col.aggregatorid === 'MAX') {
 						return (
-							pre +
-							"g['" +
-							colas +
-							"']=Math.max(g['" +
-							colas +
-							"']," +
-							colexp +
-							');' +
-							post
+							pre + 'if ((y=' + colexp + ") > g['" + colas + "']) g['" + colas + "'] = y;" + post

```

Comment:
I used `y` here assuming it is intended to be a temporary variable (as far as I can tell from how it is used in `AVG`)